hw/ip_templates/gpio/dv/env/seq_lib/gpio_base_vseq.sv.tpl

```diff
@@ -124,15 +124,16 @@ class ${module_instance_name}_base_vseq extends cip_base_vseq #(
     end
   endtask : pgm_intr_regs
 
-  // Wait a few cycles. If force_positive is true, Wait at least one clock cycle.
+  // Wait a few cycles. If force_positive is true, wait at least one clock cycle.
+  // If a reset is detected wait for the reset process is done and return immediately.
```

Comment:
This second line is not correct. If you're not sure why, run with waves enabled and look to see when this task finishes.